native/cocos/renderer/gfx-base/GFXDef-common.h

```diff
@@ -1212,48 +1219,45 @@ struct InputState {
     AttributeList attributes;
 };
 
-// Use uint32_t for all boolean values to convert memory to RasterizerState* in shared memory.
```

Comment:
Is there any reason to delete the comments?

Reply:
I thought we don't have the shared memory system anymore

Reply:
It is needed for performance, as the states are checked and modified in 2D components.

Reply:
The states are checked and modified many times every frame.